autoload/projectionist.vim

```diff
@@ -851,7 +852,7 @@ function! s:edit_command(mods, edit, count, ...) abort
       let i = 0
       for [alt, _] in alternates
         let i += 1
-        let relative = fnamemodify(alt, ":~:.")
+        let relative = substitute(fnamemodify(alt, ":~:."), "^\./", "", "")
```

Comment:
Might have to use `projectionist#slash()` in the search pattern here instead of literal `/`; not sure.

Reply:
Did that, provisionally, in another branch (in fa95a77153369086e0ab926d50efd4cf1b9b43af).